plugins/tekton/tests/tekton.spec.ts

```diff
@@ -0,0 +1,319 @@
+import { expect, Locator, Page, test } from '@playwright/test';
+
+test.describe('Tekton plugin', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('http://localhost:3000/tekton');
```

Comment:
It would be awesome if this could be overridden via an environment variable.

Reply:
added PLUGIN_BASE_URL to the config